hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: patch
+
+This change ensures that the correct error (InvalidArgument) is raised when
```

Comment:
I missed this in the in-person review and it's totally my fault, but this patch note is not quite right.  Instead, can you change it to read as follows?

```
This patch handles passing an empty :class:`python:enum.Enum` to
:func:`~hypothesis.strategies.from_type` returns 
:func:`~hypothesis.strategies.nothing`, instead of raising an 
internal :class:`python:AssertionError`.
```